families/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Cargill Incorporated
+# Copyright 2018 Intel Corporation
```

Comment:
Since you are not adding a new file but rewritting an old one the way we have handled Copyright in the past has been to add the new copyright below the previous for example

# Copyright 2018 Cargill Incorporated
# Copyright 2018 Intel Corporation

Reply:
We replaced that file with a new one. We've replaced everything in the families directory, except the "tests" directory.
Does the rule above still apply?